### ISSUE TITLE
Works great! How about widths?

### ISSUE THREAD

**Issue by dolbex** [guanzo/vue-smooth-height#5]
Could be super useful to do both.

**Comment by guanzo** [comment]
This mixin originally wanted to transition any css property. I eventually decided to focus on height because it was by far the most affected property when an element is added to the DOM. I agree that transitioning widths would be useful, but it obviously conflicts with the name, vue-smooth-height. If I ever decide to implement widths, or receive a pull request, I'll need to create another package, probably vue-smooth-element.

Long story short, widths is a good idea but not immediately in the works.

**Comment by dolbex** [comment]
Cool, thanks for the explanation!
